mautic/mautic
# Japanesse fonts in forms

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:

Form results emails don't cooperate with template emails using Japanese fonts


#### Steps to test this PR:

- Create a template email
- Set the default font using the Japanese characters, add text TEST
- Save the email
- Create a form with just an email field (add other fields if you want, but they're not necessary)
- Add a form action to send form results
- Select the email you just created to use as a template for the form results email
- Click save/update (you even do not need to save it, it is broken in the session)
- Edit the send form results action

Expected results: The basic text TEST should still appear as what would be sent when a contact submits the form
